RELEASING.md

```diff
@@ -0,0 +1,204 @@
+# Release Documentation
+
+This document describes the process on how to release a new version of Valkey-GLIDE, how to use the GLIDE's CD workflows, and how to release a new wrapper.
+
+## GLIDE - How to release a new version
+
+**Contact a Valkey-GLIDE maintainer** on how to trigger the CD workflows.
```

Comment:
This is repeated below when CD workflows are mentioned.  Maybe we don't need this line anymore?